### ISSUE TITLE
Contact photo in token doesn't show

### ISSUE THREAD

**Issue by daquexian** [k9mail/k-9#2337]
### Expected behavior
When composing email, the contact photo in token shows

### Actual behavior
The contact photo in token doesn't show at the beginning, but becomes shown if continue to type.

![photo6186176924578064303](https://cloud.githubusercontent.com/assets/11607199/23577281/f1cb2dc2-00f5-11e7-94df-ddfbcee0afb0.jpg)

### Steps to reproduce
1. Create a contact with photo
2. Compose a new email
3. Click the drop down item of the contact with photo
4. No photo shows in the token
5. Continue to type in the input box, the photo shows.

### Environment
K-9 Mail version: 5.203

Android version: 7.1.1, and I reproduced in API 23 emulator

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

**Comment by daquexian** [comment]
This issue seems related to Glide (See [this](https://github.com/bumptech/glide/issues/395)). I used `override` method of Glide and this issue fixed partically.

Since the related code is modified in my another PR, #2315 , I want to fix this issue myself after that PR is approved or canceled :) Is it ok? @cketti @Valodim @philipwhiuk

**Comment by philipwhiuk** [comment]
Yeah it's definitely not helped by us using two different loaders. I agree that we should land your PR and then look at it - two people changing the same code just leads to bugs and merge-hell.

There's also some code in `ContactPictureLoader` which might be relevant but I've not looked too much into it.

```
            try {
                task.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
            } catch (RejectedExecutionException e) {
                // We flooded the thread pool queue... use a fallback picture
                imageView.setImageBitmap(calculateFallbackBitmap(address));
            }
```

Also, I note that we are using Glide 3.6.1 instead of 3.7.0. If we can upgrade we probably should because there's some bugfixes we might want.

**Comment by daquexian** [comment]
This issue is fixed by my PR #2315. So I close it :)
